GetandTake/Areas/Identity/Pages/Account/ConfirmEmailChange.cshtml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Text;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.WebUtilities;
+
+namespace GetandTake.Areas.Identity.Pages.Account;
+
+public class ConfirmEmailChangeModel : PageModel
+{
+    private readonly UserManager<IdentityUser> _userManager;
+    private readonly SignInManager<IdentityUser> _signInManager;
+
+    public ConfirmEmailChangeModel(
+        UserManager<IdentityUser> userManager, 
+        SignInManager<IdentityUser> signInManager)
+    {
+        _userManager = userManager;
+        _signInManager = signInManager;
+    }
+
+    [TempData]
+    public string StatusMessage { get; set; }
+
+    public async Task<IActionResult> OnGetAsync(string userId, string email, string code)
+    {
+        if (string.IsNullOrWhiteSpace(userId)|| string.IsNullOrWhiteSpace(email) || string.IsNullOrWhiteSpace(code))
```

Comment:
Fir such long conditions I advise you to put them into variable, wrap into a column and put this variable into condition. It will look easier and does not require wrapping for small devices screens. So, the final result will look like:
```
        var isAnyRequiredParamMissing =
                   string.IsNullOrWhiteSpace(userId) ||
                   string.IsNullOrWhiteSpace(email) ||
                   string.IsNullOrWhiteSpace(code);

        if (isAnyRequiredParamMissing) { ... }
```